gremlin-dotnet/src/Gremlin.Net/Process/Traversal/DefaultTraversal.cs

```diff
@@ -198,9 +198,10 @@ public bool HasNext()
         /// </summary>
         /// <returns>The result.</returns>
         public E Next()
-        {
-            MoveNext();
-            return Current;
+        { 
+            return MoveNext() 
```

Comment:
Small nit: this should probably be one line to follow the style that exists in our cs files.

Reply:
Updated this to follow the style on the rest of the files